Makefile.PL

```diff
@@ -15,7 +15,7 @@ use Term::ANSIColor;
 if (!$ENV{CI}) { # don't regenerate model in CI until we figure out why the default namespace isn't omdoc
   my @args = ("bash", "-c", "type trang || which trang ||echo '\n\033[0;31mTrang is not found which could lead to a make failure.\nYou can ignore this message if you do not intend to change the schema.\nSee README for more info!\n \033[0m' ");
   my $ifTrang = system(@args);
-  my $updateModel = `cd lib/LaTeXML/resources/RelaxNG/; make; make distclean`;
+  my $updateModel = `cd lib/LaTeXML/resources/RelaxNG/; make`;
```

Comment:
This line automatically generates the `.rng`s whenever `perl Makefile.PL` is called. I removed the `distclean` here because the `rng`s are required by the tests.

There are two things I don't like about it:
- Anytime we call `perl Makefile.PL` they are regenerated -- even when the source files didn't change or `trang` isn't installed. I guess we should move it into a seperate (main) `Makefile` target. 
- The `rng`s are currently committed, a lot of those are identical to the LaTeXML counterparts. Not sure if there is a smarter way -- @dginev?

Reply:
see https://github.com/slatex/LaTeXML-Plugin-sTeX/pull/143#issuecomment-564923374